browser/src/Editor/OniEditor/OniEditor.tsx

```diff
@@ -145,7 +146,10 @@ export class OniEditor implements IEditor {
             wrapReactComponentWithLayer("oni.layer.errors", <ErrorsContainer />),
         )
 
-        this._neovimEditor.bufferLayers.addBufferLayer("image", buf => new ImageBufferLayer(buf))
+        this._neovimEditor.bufferLayers.addBufferLayer(
+            buf => [".gif", ".jpg", ".jpeg", ".bmp", ".png"].includes(path.extname(buf.filePath)),
```

Comment:
Could we make this a configuration setting? Maybe `editor.imageLayerExtensions`? That way, users can always override it or add new extensions.

Reply:
Just pushed the change 👍